VCU/Phantom/main.c

```diff
@@ -64,12 +64,38 @@ void phantomDriversInit()
      MCP48FV_Init();             // Initialize DAC Library
 }
 
+void phantomQueueInit()
+{
+    QueueHandle_t statusQ = Phantom_createQueue(10, sizeof(int)),
+                  stateThrottleQ = Phantom_createQueue(1, sizeof(int)),
+                  stateInterruptQ = Phantom_createQueue(1, sizeof(int)),
```

Comment:
Let's explicitly name mailboxes so we know these handles don't have to be synchronized